Doc/howto/clinic.rst

```diff
@@ -990,28 +991,28 @@ Consider the following example:
 
     foo: Py_ssize_t = sys.maxsize - 1
 
-``sys.maxsize`` can have different values on different platforms.  Therefore
+:py:attr:`sys.maxsize` can have different values on different platforms.  Therefore
 Argument Clinic can't simply evaluate that expression locally and hard-code it
 in C.  So it stores the default in such a way that it will get evaluated at
 runtime, when the user asks for the function's signature.
 
 What namespace is available when the expression is evaluated?  It's evaluated
 in the context of the module the builtin came from.  So, if your module has an
-attribute called "``max_widgets``", you may simply use it:
+attribute called :py:attr:`!max_widgets`, you may simply use it:
 
 .. code-block:: none
 
     foo: Py_ssize_t = max_widgets
 
 If the symbol isn't found in the current module, it fails over to looking in
-``sys.modules``.  That's how it can find ``sys.maxsize`` for example.  (Since you
+:py:attr:`sys.modules`.  That's how it can find :py:attr:`sys.maxsize` for example.  (Since you
 don't know in advance what modules the user will load into their interpreter,
 it's best to restrict yourself to modules that are preloaded by Python itself.)
 
 Evaluating default values only at runtime means Argument Clinic can't compute
 the correct equivalent C default value.  So you need to tell it explicitly.
 When you use an expression, you must also specify the equivalent expression
-in C, using the ``c_default`` parameter to the converter:
+in C, using the *c_default* parameter to the converter:
```

Comment:
Parameters are usually marked up with pairs of \*, both for C functions and Python functions/methods. I think we should use the same markup for Argument Clinic converters as well, since they are in practice Python functions.

Reply:
Then please update also for other parameters, like `subclass_of`.

Reply:
I've marked up what I could find in d96fdebe8f252464eac9bfa6582eed29c54f9f30. I might have missed some, though.